package.json

```diff
@@ -31,7 +31,7 @@
     "@readme/emojis": "^2.1.0",
     "@readme/ui": "^1.17.6",
     "babel-polyfill": "^6.26.0",
-    "oas": "^9.0.0",
+    "oas": "github:readmeio/oas#feat/jsonschema-examples",
```

Comment:
Note to self: bump this after https://github.com/readmeio/oas/pull/358 is merged and published.